modules/sdk-coin-osmo/test/unit/transactionBuilder/StakingActivateBuilder.ts

```diff
@@ -0,0 +1,143 @@
+    should.equal(rawTx, testTx.signedTxBase64);
+    should.deepEqual(tx.inputs, [
+      {
+        address: testTx.delegator,
+        value: testTx.sendMessage.value.amount.amount,
+        coin: 'tosmo',
+      },
+    ]);
+    should.deepEqual(tx.outputs, [
+      {
+        address: testTx.validator,
+        value: testTx.sendMessage.value.amount.amount,
+        coin: 'tosmo',
+      },
+    ]);
+  });
+
+  it('should build a Delegate tx without signature', async function () {
+    const txBuilder = factory.getStakingActivateBuilder();
+    txBuilder.sequence(testTx.sequence);
+    txBuilder.gasBudget(testTx.gasBudget);
+    txBuilder.messages([testTx.sendMessage.value]);
+    txBuilder.publicKey(toHex(fromBase64(testTx.pubKey)));
+    const tx = await txBuilder.build();
+    const json = await (await txBuilder.build()).toJson();
+    should.equal(tx.type, TransactionType.StakingActivate);
+    should.deepEqual(json.gasBudget, testTx.gasBudget);
+    should.deepEqual(json.sendMessages, [testTx.sendMessage]);
+    should.deepEqual(json.publicKey, toHex(fromBase64(testTx.pubKey)));
+    should.deepEqual(json.sequence, testTx.sequence);
+    tx.toBroadcastFormat();
+    should.deepEqual(tx.inputs, [
+      {
+        address: testTx.delegator,
+        value: testTx.sendMessage.value.amount.amount,
+        coin: 'tosmo',
+      },
+    ]);
+    should.deepEqual(tx.outputs, [
+      {
+        address: testTx.validator,
+        value: testTx.sendMessage.value.amount.amount,
+        coin: 'tosmo',
+      },
+    ]);
+  });
+
+  it('should sign a Delegate tx', async function () {
+    const txBuilder = factory.getStakingActivateBuilder();
+    txBuilder.sequence(testTx.sequence);
+    txBuilder.gasBudget(testTx.gasBudget);
+    txBuilder.messages([testTx.sendMessage.value]);
+    txBuilder.accountNumber(testTx.accountNumber);
+    txBuilder.chainId(testTx.chainId);
+    txBuilder.sign({ key: toHex(fromBase64(testTx.privateKey)) });
+    const tx = await txBuilder.build();
+    const json = await (await txBuilder.build()).toJson();
+    should.equal(tx.type, TransactionType.StakingActivate);
+    should.deepEqual(json.gasBudget, testTx.gasBudget);
+    should.deepEqual(json.sendMessages, [testTx.sendMessage]);
+    should.deepEqual(json.publicKey, toHex(fromBase64(testTx.pubKey)));
+    should.deepEqual(json.sequence, testTx.sequence);
+    const rawTx = tx.toBroadcastFormat();
+    should.equal(tx.signature[0], toHex(fromBase64(testTx.signature)));
+    should.equal(rawTx, testTx.signedTxBase64);
+    should.deepEqual(tx.inputs, [
+      {
+        address: testTx.delegator,
+        value: testTx.sendMessage.value.amount.amount,
+        coin: 'tosmo',
+      },
+    ]);
+    should.deepEqual(tx.outputs, [
+      {
+        address: testTx.validator,
+        value: testTx.sendMessage.value.amount.amount,
+        coin: 'tosmo',
+      },
+    ]);
+  });
+
+  xit('should submit a send transaction', async () => {
```

Comment:
Is there a reason why we are skipping this test case?

Reply:
for now, remove this testcase.

Reply:
+1

Reply:
these tests are similar to integration test and only supposed to be run manually to do the sanity. So skipping this test is fine as long as it is working manually. Maybe we can move it to example scripts or something